project_template/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required(VERSION 3.15)

```

Comment:
>3.15 [](start = 31, length = 4)

Are there any features really needed from 3.15?
The only reason I am saying this is that in the IoTHub SDK case there were several platforms that had lower versions of cmake (one of them had 2.8 at some point). Consider using the lowest version according to the needed features from cmake.

Reply:
We should hopefully (bare minimum) be using something above 3.0. The more recent versions of cmake make use of targets and properties rather than passing along variables.

---
In reply to: [309437127](https://github.com/Azure/azure-sdk-for-c/pull/2#discussion_r309437127) [](ancestors = 309437127)

Reply:
The current most likely to standardize on is 3.6.